src/PrestaShopBundle/Service/Grid/ResponseBuilder.php

```diff
@@ -99,4 +121,47 @@ public function buildSearchResponse(
 
         return new RedirectResponse($redirectUrl, 302);
     }
+
+    /**
+     * @param string $filterId
+     *
+     * @return void
+     */
+    private function resetPersistedFilter(string $filterId): void
+    {
+        if (empty($filterId)) {
+            return;
+        }
+        $adminFilter = $this->adminFilterRepository->findByEmployeeAndFilterId(
+            $this->employeeId,
+            $this->shopId,
+            $filterId
+        );
+        $this->adminFilterRepository->unsetFilters($adminFilter);
```

Comment:
You need to check that `$adminFilter` is not null

Reply:
Fixed